packages/admin/admin/src/common/buttons/split/SplitButton.tsx

```diff
@@ -91,8 +93,14 @@ const SplitBtn = ({
             <Popover
                 open={open}
                 anchorEl={anchorRef.current}
-                anchorOrigin={{ vertical: "bottom", horizontal: "center" }}
-                transformOrigin={{ vertical: "top", horizontal: "center" }}
+                anchorOrigin={{
+                    vertical: popoverProps?.anchorOrigin?.vertical ?? "bottom",
+                    horizontal: popoverProps?.anchorOrigin?.horizontal ?? "center",
+                }}
+                transformOrigin={{
+                    vertical: popoverProps?.transformOrigin?.vertical ?? "top",
+                    horizontal: popoverProps?.transformOrigin?.horizontal ?? "center",
+                }}
```

Comment:
Using this approach you only allow to override `anchorOrigin` and `transformOrigin`. However, `popoverProps?: Partial<PopoverProps>` indicates that I may override _all_ Popover props. This can be achieved by leaving `anchorOrigin` and `transformOrigin` as they are, and spreading `popoverProps` afterwards:

```typescript
<Popover
    open={open}
    anchorEl={anchorRef.current}
    anchorOrigin={{ vertical: "bottom", horizontal: "center" }}
    transformOrigin={{ vertical: "top", horizontal: "center" }}
    onClose={handleClose}
    {...popoverProps}
/>
```

Now, if a developer provides `anchorOrigin` and `transformOrigin` in `popoverProps`, they will be overridden (and other props too!)

Reply:
I added your suggested improvements to the code and updated the storyboard.